content/1_docs/1_guide/20_kirby-and-privacy/guide.txt

```diff
@@ -63,6 +63,10 @@ No other personal or site data is transmitted to our server.
 
 Your sites may store or process additional personal data depending on the Kirby plugins and custom code you are using. For example, some plugins like contact form plugins also use sessions for technical reasons. Themes may include web fonts, external scripts or tracking code.
 
+### Usage of external images, embeds or other remote data
+
+When integrating things into your site from other servers, there is always the possibility to reveal information about your visitors (e.g. IP address) to third parties – or even to load third-party cookies or other scripts that e.g. could track visitors across the web. Reflect on this risk when using e.g. external images, the `video` KirbyTag or other embeds and act accordingly.
```

Comment:
Two minor details:

- I‘d replace “things” with “resources”
- The sentence contains a lot of `e.g.`s. Maybe you can reword some of those.